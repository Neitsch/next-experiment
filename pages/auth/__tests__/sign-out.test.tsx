import { shallow } from "enzyme";
import * as React from "react";

describe("Sign In", () => {
  beforeEach(() => {
    jest.resetModules();
  });
  it("renders", () => {
    const logout = jest.fn();
    const unsetToken = jest.fn();
    jest.doMock("../../../lib/auth/lock", () => ({
      logout,
    }));
    jest.doMock("../../../lib/auth/index", () => ({
      unsetToken,
    }));
    const SignOut = require("../sign-out").default;
    const rendered = shallow(<SignOut />);
    expect(rendered).toMatchSnapshot();
    expect(logout).toHaveBeenCalled();
    expect(unsetToken).toHaveBeenCalled();
  });
});
