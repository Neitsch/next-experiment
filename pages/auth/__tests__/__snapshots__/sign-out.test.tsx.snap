// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign In renders 1`] = `""`;
